apps/hobbies-helsinki/next.config.js

```diff
@@ -0,0 +1,311 @@
+// @ts-check
+
+// https://nextjs.org/docs/api-reference/next.config.js/introduction
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+const { withSentryConfig } = require('@sentry/nextjs');
+const pc = require('picocolors');
+const packageJson = require('./package.json');
+const { i18n } = require('./next-i18next.config');
+
+// const enableCSP = true;
+const trueEnv = ['true', '1', 'yes'];
+
+const isProd = process.env.NODE_ENV === 'production';
+const isCI = trueEnv.includes(process.env?.CI ?? 'false');
+
+const NEXTJS_IGNORE_ESLINT = trueEnv.includes(
+  process.env?.NEXTJS_IGNORE_ESLINT ?? 'false'
+);
+const NEXTJS_IGNORE_TYPECHECK = trueEnv.includes(
+  process.env?.NEXTJS_IGNORE_TYPECHECK ?? 'false'
+);
+const NEXTJS_DISABLE_SENTRY = trueEnv.includes(
+  process.env?.NEXTJS_DISABLE_SENTRY ?? 'false'
+);
+const NEXTJS_SENTRY_UPLOAD_DRY_RUN = trueEnv.includes(
+  process.env?.NEXTJS_SENTRY_UPLOAD_DRY_RUN ?? 'false'
+);
+const NEXTJS_SENTRY_DEBUG = trueEnv.includes(
+  process.env?.NEXTJS_SENTRY_DEBUG ?? 'false'
+);
+const NEXTJS_SENTRY_TRACING = trueEnv.includes(
+  process.env?.NEXTJS_SENTRY_TRACING ?? 'false'
+);
+
+/**
+ * A way to allow CI optimization when the build done there is not used
+ * to deliver an image or deploy the files.
+ * @link https://nextjs.org/docs/advanced-features/source-maps
+ */
+const disableSourceMaps = trueEnv.includes(
+  process.env?.NEXT_DISABLE_SOURCEMAPS ?? 'false'
+);
+
+if (disableSourceMaps) {
+  console.warn(
+    `${pc.yellow(
+      'notice'
+    )}- Sourcemaps generation have been disabled through NEXT_DISABLE_SOURCEMAPS`
+  );
+}
+
+if (NEXTJS_SENTRY_DEBUG) {
+  console.warn(
+    `${pc.yellow(
+      'notice'
+    )}- Build won't use sentry treeshaking (NEXTJS_SENTRY_DEBUG)`
+  );
+}
+
+// Tell webpack to compile those packages
+// @link https://www.npmjs.com/package/next-transpile-modules
+const tmModules = ['react-helsinki-headless-cms', 'events-helsinki-components'];
```

Comment:
next-app example project has the transpiled modules like this:
```
const tmModules = [
  // for legacy browsers support (only in prod)
  ...(isProd
    ? [
        // ie: '@react-google-maps/api'...
        'ky', // does not pass es-2017 checks
        'events-helsinki-components',
        'react-helsinki-headless-cms',
      ]
    : []),
   ```
   This probably avoid unnecessary transpiling on dev server

Reply:
if the components are not transpiled, an error from sass-css will occure. There might be a problem there, but something more is needed than this

Reply:
transpiling is always needed for sass